go.mod

```diff
@@ -1,15 +1,13 @@
 module github.com/mattermost/mmctl
 
-go 1.12
+go 1.13
 
 require (
-	github.com/cpuguy83/go-md2man v1.0.10 // indirect
-	github.com/golang/mock v1.2.0
-	github.com/magefile/mage v1.8.0
+	github.com/golang/mock v1.3.1
+	github.com/magefile/mage v1.9.0
```

Comment:
Why is `mage` part of `go.mod`? We are not importing it anywhere.

Reply:
It is being imported inside `magefile.go`, so is being taking into account by the vendor command

Reply:
I see :+1: